### DISCUSSION TITLE
Bind refresh rate to power plans

### DISCUSSION THREAD

**Discussion by utchka** [seerge/g-helper#1147] [Ideas]
Hello everyone!
I usually keep refresh rate at 60Hz and silent mode selected to save energy, then switch to 240Hz/turbo mode before launching a game. 
I would like to set a specific refresh rate for each power mode, for example: silent/60Hz, turbo/240Hz.
What do you think?

**Comment by seerge** [comment]
@utchka why would you use 60Hz when you are on charger in a first place ? :)

**Reply by utchka** [reply]
Just to save a little more energy. I don't really need a high refresh rate for youtube videos and emails. Electricity bills almost doubled here in Italy in less than a year.

**Reply by seerge** [reply]
I don’t think that there is any realistically measurable difference between refresh rates :) we speak about 0.5w max :))

So for 10 hours a day, 30 days in a month we speak about 150 Wh or in (high!) Dutch prices (30 cents per kWh) - 4.5 euro cents per moth

**Reply by XyKER1** [reply]
It would be useful also to disable screen overdrive automatically when switching to battery as part of the auto feature, as this uses more power.

**Reply by utchka** [reply]
I know it's not a big thing, anyway I see a 2W difference when switching from 240Hz to 60Hz. So I would save 18 cents a month! :-D
